doc/getting-started/getting-started.rst

```diff
@@ -368,7 +360,7 @@ Generate a Board Configuration File
    .. code-block:: bash
 
       cd ~/acrn-work
-      sudo board_inspector.py my_board
+      sudo board_inspector my_board
```

Comment:
This command fails for me:
```
acrn@acrn-Vecow:~/acrn-work$ sudo board_inspector my_board
[sudo] password for acrn: 
sudo: board_inspector: command not found

```
Where is the ``board_inspectot`` program found?  Here's my PATH:
```
echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin
```

Reply:
Looking around, I found the actual name of the tool has changed from ``board_inspector`` to ``acrn-board-inspector`` so I tried running that, and got this failure now.  Looks like there are some missing required packages on the target system after installing the board inspector debian package:
```
acrn@acrn-Vecow:~/acrn-work$ sudo acrn-board-inspector my_board
[sudo] password for acrn: 
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3/dist-packages/acrn_board_inspector/board_inspector.py", line 15, in <module>
    from tqdm import tqdm
ModuleNotFoundError: No module named 'tqdm'
```

Reply:
Reading ahead, it looks like packages such as ``tqdm`` may be installed as part of the acrn-hypervisor Debian packages, but these are installed AFTER we run the acrn-board-inspector so it's too late. @tttech-industrial-buchsbaum am I right?

Reply:
@dbkinder yes, the program name should be acrn-board-inspector.
For the tqdm issue, can I add the dependent package python3-tqdm for python3-acrn-board-inspector into debian/control ? I have tested that the issue should be gone after this change and rebuilding board-inspector. @dbkinder @tttech-industrial-buchsbaum

Reply:
Have you verified that ``tqdm`` is the only missing dependency, or are there more?

Reply:
Yes, I have verified that `tqdm` is the only  missing dependency. After I added the dependent package python3-tqdm for python3-acrn-board-inspector into debian/control and installed the rebuilt python3-acrn-board-inspector_3.2-unstable~1.gbp6d48d4_all.deb, the execution log as below:
```
acrn@acrn-tgl-vecow-04:~/acrn-work$ sudo apt install -y /tmp/python3-acrn-board-inspector_3.2-unstable~1.gbp6d48d4_all.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'python3-acrn-board-inspector' instead of '/tmp/python3-acrn-board-inspector_3.2-unstable~1.gbp6d48d4_all.deb'
The following additional packages will be installed:
  cpuid msr-tools python3-bs4 python3-elementpath python3-html5lib python3-lxml python3-soupsieve python3-tqdm python3-webencodings
  python3-xmlschema
Suggested packages:
  acrn-doc python3-genshi python-lxml-doc
The following NEW packages will be installed:
  cpuid msr-tools python3-acrn-board-inspector python3-bs4 python3-elementpath python3-html5lib python3-lxml python3-soupsieve
  python3-tqdm python3-webencodings python3-xmlschema
0 upgraded, 11 newly installed, 0 to remove and 1 not upgraded.
Need to get 1,864 kB/1,961 kB of archives.
After this operation, 8,930 kB of additional disk space will be used.
Get:1 /tmp/python3-acrn-board-inspector_3.2-unstable~1.gbp6d48d4_all.deb python3-acrn-board-inspector all 3.2-unstable~1.gbp6d48d4 [97.1 kB]
Get:2 http://cn.archive.ubuntu.com/ubuntu jammy/universe amd64 cpuid amd64 20211210-1 [104 kB]
Get:3 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 msr-tools amd64 1.3-4 [10.3 kB]
Get:4 http://cn.archive.ubuntu.com/ubuntu jammy/universe amd64 python3-tqdm all 4.57.0-2 [93.0 kB]
Get:5 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-lxml amd64 4.8.0-1build1 [1,150 kB]
Get:6 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-elementpath all 2.5.0-1 [123 kB]
Get:7 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-xmlschema all 1.10.0-1 [172 kB]
Get:8 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-soupsieve all 2.3.1-1 [33.0 kB]
Get:9 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-bs4 all 4.10.0-2 [79.1 kB]
Get:10 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-webencodings all 0.5.1-4 [11.8 kB]
Get:11 http://cn.archive.ubuntu.com/ubuntu jammy/main amd64 python3-html5lib all 1.1-3 [87.0 kB]
Fetched 1,864 kB in 1s (1,586 kB/s)
Selecting previously unselected package cpuid.
(Reading database ... 200268 files and directories currently installed.)
Preparing to unpack .../00-cpuid_20211210-1_amd64.deb ...
Unpacking cpuid (20211210-1) ...
Selecting previously unselected package msr-tools.
Preparing to unpack .../01-msr-tools_1.3-4_amd64.deb ...
Unpacking msr-tools (1.3-4) ...
Selecting previously unselected package python3-tqdm.
Preparing to unpack .../02-python3-tqdm_4.57.0-2_all.deb ...
Unpacking python3-tqdm (4.57.0-2) ...
Selecting previously unselected package python3-lxml:amd64.
Preparing to unpack .../03-python3-lxml_4.8.0-1build1_amd64.deb ...
Unpacking python3-lxml:amd64 (4.8.0-1build1) ...
Selecting previously unselected package python3-elementpath.
Preparing to unpack .../04-python3-elementpath_2.5.0-1_all.deb ...
Unpacking python3-elementpath (2.5.0-1) ...
Selecting previously unselected package python3-xmlschema.
Preparing to unpack .../05-python3-xmlschema_1.10.0-1_all.deb ...
Unpacking python3-xmlschema (1.10.0-1) ...
Selecting previously unselected package python3-acrn-board-inspector.
Preparing to unpack .../06-python3-acrn-board-inspector_3.2-unstable~1.gbp6d48d4_all.deb ...
Unpacking python3-acrn-board-inspector (3.2-unstable~1.gbp6d48d4) ...
Selecting previously unselected package python3-soupsieve.
Preparing to unpack .../07-python3-soupsieve_2.3.1-1_all.deb ...
Unpacking python3-soupsieve (2.3.1-1) ...
Selecting previously unselected package python3-bs4.
Preparing to unpack .../08-python3-bs4_4.10.0-2_all.deb ...
Unpacking python3-bs4 (4.10.0-2) ...
Selecting previously unselected package python3-webencodings.
Preparing to unpack .../09-python3-webencodings_0.5.1-4_all.deb ...
Unpacking python3-webencodings (0.5.1-4) ...
Selecting previously unselected package python3-html5lib.
Preparing to unpack .../10-python3-html5lib_1.1-3_all.deb ...
Unpacking python3-html5lib (1.1-3) ...
Setting up python3-tqdm (4.57.0-2) ...
Setting up msr-tools (1.3-4) ...
Setting up cpuid (20211210-1) ...
Setting up python3-webencodings (0.5.1-4) ...
Setting up python3-elementpath (2.5.0-1) ...
Setting up python3-html5lib (1.1-3) ...
Setting up python3-xmlschema (1.10.0-1) ...
Setting up python3-lxml:amd64 (4.8.0-1build1) ...
Setting up python3-soupsieve (2.3.1-1) ...
Setting up python3-bs4 (4.10.0-2) ...
Setting up python3-acrn-board-inspector (3.2-unstable~1.gbp6d48d4) ...
Processing triggers for man-db (2.10.2-1) ...
acrn@acrn-tgl-vecow-04:~/acrn-work$ sudo acrn-board-inspector my_board
Generating board XML my_board. This may take a few minutes...
 70%|████████████████████████████████████████████████████████████████████▌                             | 70/100 [00:18<00:06,  4.48it/s]2022-11-11 19:31:42-root-WARNING:-The PCI device at 00:02.0 has SR-IOV capability which is currently disabled.
2022-11-11 19:31:42-root-WARNING:-If that device is intended to provide virtual functions in your scenario, write (as root) the number of VFs to /sys/devices/pci0000:00/0000:00:02.0/sriov_numvfs and rerun the board inspector.
========================================================================================================================
WARNING
These issues affect optional features. You can ignore them if they don't apply to you.

2022-11-11 19:31:42-root-WARNING:-The PCI device at 00:02.0 has SR-IOV capability which is currently disabled.
2022-11-11 19:31:42-root-WARNING:-If that device is intended to provide virtual functions in your scenario, write (as root) the number of VFs to /sys/devices/pci0000:00/0000:00:02.0/sriov_numvfs and rerun the board inspector.
========================================================================================================================
SUCCESS: Board configuration file my_board.xml generated successfully and saved to /home/acrn/acrn-work

130it [00:20,  6.34it/s]
```

Reply:
Great. Can you submit a PR with that change and I'll give it a try and continue the GSG review.

Reply:
@tttech-industrial-buchsbaum has submitted in PR: https://github.com/projectacrn/acrn-hypervisor/pull/8306/commits/4d13c5cb04a52042c4da634ee4473dff23ff5812

Reply:
#8306 is not merged yet so I can't verify it fixes the GSG problem

Reply:
@dbkinder , PR#8306 was merged.